api-reference/api-reference-template.md

```diff
@@ -0,0 +1,204 @@
+# API Reference
+
+{Before using this template, read [the accompanying guide](api-reference-guide.md) to this template}.
+
+
+## Overview
+
+This reference describes the Application Program Interfaces (APIs) that help you {access | customize | program} {product}'s {features | functionality}.
+
+### Base URL
+
+The APIs referenced in this documentation use the following base URL:
+
+```
+https://api.example.com
+```
+
+
+### Authorization
+
+All requests to the API must be authenticated and authorized before they can access or manage resources on the {environment | platform}.
+
+The {product} API uses the `{auth_type}` authorization type.
+
+You can authenticate your requests by using {authentication type and method}.
+
+For example:
+
+```
+{EXAMPLE}
```

Comment:
It would be helpful for the author to show an actual example request, say generated in Postman. Something like:
```
GET /my-endpoint
Content-Type: application/json
Authorization: Basic 123456789frefr=
Host: my-url.com
```